permit.md

```diff
@@ -1,19 +1,16 @@
 # Overview
-Tables: 
-
-* VPS_VESSEL - Holds vessel characteristic information (horsepower, length, gross tonnage) associated with a particular vessel permit 
-* VPS_FISHERY_NER - Detailed information on a vessel’s associated permit types.  
-Plan is the fishery management plan (FMP).  Cat is the category within that Plan.
-* BUS_OWN -Contains business ownership information that describest the people that own a fishing firm.  Allows for grouping of permits into firms based on identical owners.
-* VPS_OWNER -  Contains ownership information, including a business ID.
-* DEALER - Contains information on federal dealer permit application information 
 
 
-*the BUS_OWN table is not the main view for business onwer information. THat is found here: Client.bus_own@garfo_nefsc. For more information on this system see Client.md
+The following views can be found in the NEFSC_GARFO schema on nefscdb1
```

Comment:
AMS_ALLOCATION_TX
AMS_DAS_ACTIVITY_CODES
AMS_LEASE_EXCH_APPLIC
AMS_OBSERVER_TRIP
AMS_TRIP
AMS_TRIP_AND_CHARGE
AMS_TRIP_LANDING
APSD_T_SSB_CATCH_CURRENT
APSD_T_SSB_DISCARD_CURRENT
APSD_T_SSB_TRIP_CURRENT
CATCH
CFDBS_AREA
CFDBS_LOC
CFDBS_LOC2AREAS
CFDBS_SPECIES_ITIS_NE
CFDBS_STATENM
CFDBS_VALID_PORTS
CFDERS_ALL_YEARS
CLIENT_BUS_OWN
DOCUMENT
IMAGES
JOPS_OPERATOR
MQRS_MORT_ELIG_CRITERIA
PERMIT_DEALER
PERMIT_OWNER_HISTORY_PIDS
PERMIT_VALID_FISHERY
PERMIT_VPS_FISHERY_NER
PERMIT_VPS_OWNER
PERMIT_VPS_VESSEL
SCBI_SPECIES_ITIS_NE
SECTOR_ACE_MRI_AS_POUNDS
SECTOR_MRI_PERMIT_VESSEL_HISTORY
SECTOR_PSC_MRI_AS_PERCENT
SECTOR_TRANSFERS
SECTOR_TRANSFER_STOCK
SFCLAM_SFOQVR
TRIP_REPORTS_CATCH
TRIP_REPORTS_DOCUMENT
TRIP_REPORTS_IMAGES
VTR_DEALR
VTR_VLGEAR
VTR_VLSPPTBL

Reply:
I don't want to put this information into the permit.md file.

Reply:
This is going to be a giant pain in the ass if we want to keep our metadata organized by schema. -- because then everything should go into a single "GARFO_NEFSC" file. I don't want to do that, because it's just gross.

Reply:
@SWerner2  -- okay, I have created an issue based on this comment.